docs/requirements.yaml

```diff
@@ -0,0 +1,9 @@
+name: docs
+channels:
+dependencies:
+    # Base depends
+  - python=3.10
```

Comment:
Pin it to a python version so it would not break just due to some python update.